docs/guides/presearch.md

```
@@ -1,11 +1,13 @@
 # Presearch
 
-> **Category:** DePIN | **Status:** Active
+> **Category:** DePIN | **Status:** Dead
 > **Website:** [https://presearch.com](https://presearch.com)
 
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)